### PR TITLE
8.0 ddt type multy company

### PR DESCRIPTION
**Pull request by OpenCode**
In MultiCompany is useful to have different sequences for the some document. DDT sequence is defined on DDT Type. So, ddt type would be multi company compatible.

### CODE REVIEW

**Comment by zeroincombenze** [comment]
Agree
:+1:

File: l10n_it_ddt/views/stock_picking_package_preparation.xml [hunk 1]
```diff
@@ -123,6 +123,7 @@
                 <group>
                     <field name="name"/>
                     <field name="sequence_id"/>
+                    <field name="company_id"/>
```
Comment (eLBati) [inline l10n_it_ddt/views/stock_picking_package_preparation.xml]: Aggiungerei ```groups="base.group_multi_company"```

File: l10n_it_ddt/security/security.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record model="ir.rule" id="stock_ddt_type_comp_rule">
+        <field name="name">Stock DDT Typle multi-company</field>
```
Comment (eLBati) [inline l10n_it_ddt/security/security.xml]: Type

File: l10n_it_ddt/security/security.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        <field name="model_id" ref="model_stock_ddt_type"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+
+</data>
+</openerp>
```
Comment (tafaRU) [inline l10n_it_ddt/security/security.xml]: Please add newline

**Comment by eLBati** [comment]
2 remarks, then :+1: , thanks

File: l10n_it_ddt/views/stock_picking_package_preparation.xml (hunk 1 above)
Comment (OpenCode) [inline l10n_it_ddt/views/stock_picking_package_preparation.xml]: Done

File: l10n_it_ddt/security/security.xml (hunk 2 above)
Comment (OpenCode) [inline l10n_it_ddt/security/security.xml]: Typo. Sorry :) Done

File: l10n_it_ddt/security/security.xml (hunk 3 above)
Comment (OpenCode) [inline l10n_it_ddt/security/security.xml]: In xml, too? Done

**Comment by paolovalier** [comment]
tested on runbot +1

**Pull request merged**